hivemq4/base-image/entrypoints.d/30_configure_permissions.sh

```diff
@@ -4,7 +4,7 @@
 if [[ "$1" = "/opt/hivemq/bin/run.sh" && "$(id -u)" = '0' && "${HIVEMQ_NO_ROOT_STEP_DOWN}" != "true" ]]; then
     uid="hivemq"
     gid="hivemq"
-    exec_cmd="exec gosu hivemq"
+    exec_cmd="exec chroot --skip-chdir --userspec=hivemq /"
```

Comment:
did we test that signals are properly propagated to the hivemq process?

Reply:
do you have a particular test in mind? I have tried it with SIGINT and the hivemq process terminates properly.